lib/cc/engine/analyzable_files.rb

```diff
@@ -19,7 +18,7 @@ def all
       def fetch_files(paths)
         paths.map do |path|
           if path =~ %r{/$}
-            Dir.glob("#{path}/**/*.coffee")
+            Dir.glob("#{path}**/*.coffee")
```

Comment:
I'll update the description to describe both issues in more detail instead of lumping them into one.